apps/omg_watcher/test/support/integration/test_helper.ex

```diff
@@ -66,10 +66,17 @@ defmodule OMG.Watcher.Integration.TestHelper do
   @doc """
   We need to wait on both a margin of eth blocks and exit processing
   """
-  def wait_for_exit_processing(exit_eth_height, timeout \\ 5_000) do
+  def wait_for_exit_processing(exit_eth_height, process_exits_args, timeout \\ 5_000) do
```

Comment:
these functions were intended to just wait, now you're waiting for the exit processing (as in `ExitProcessor`) and then calling `processExit` (different meaning of"process" here) in the contract. 

Now, the behavior is surprising (you modify the contract and SUT!).

Please undo and solve separately and test more explicitly in the integration test you've modified:

```
...
startExit
wait_for_exit_processing
check get_utxos
processExit
wait some more using a different specific function (wait_for_exit_finalization_processing?)
check get_utxos and get_exitable_utxos
```

there's just a single occurrence of this so no need to over-generalize. The other test modified can be reverted to old state, thanks to the old meaning of `wait_for_exit_processing`